tests/acceptance/features/bootstrap/TagsContext.php

```diff
@@ -1671,6 +1676,63 @@ public function userSearchesForTagUsingWebDavAPI($user, $tagName) {
 		$this->featureContext->setResponseXmlObject($responseXmlObject);
 	}
 
+	/**
+	 * @When user :user searches for following tag using the webDAV API
+	 *
+	 * @param string $user
+	 * @param TableNode $tagNames
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userSearchesForFollowingTagsUsingWebDAVApi($user, TableNode $tagNames) {
+		$this->searchForTagsOfFileWithReportUsingWebDAVApi(
+			$user, $tagNames
+		);
+	}
+
+	/**
+	 * @When user :user searches for tag :tagName using the webDAV API
```

Comment:
```suggestion
	 * @When user :user searches for resources tagged with tag :tagName using the webDAV API
```
Maybe we should add these words to the step text to make it say exactly what the user is searching for?